client/SDL3/dialogs/res/CMakeLists.txt.orig

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
This file should not be part of the PR ?

Reply:
removed.